### ISSUE TITLE
Retrieve archived entries

### ISSUE THREAD

**Issue by Freddyvdh** [patricoos/PortaCapena.OdooJsonRpcClient#42]
Hi,

is it possible to retrieve archived entries ie. users with the RPC client?

Thanks
Freddy

**Comment by manitra** [comment]
On the UI, you can get archived entries by adding a `active = false` filter.
Did you try to do the same using the API ?

**Comment by AArmandoPerez** [comment]
Hello @Freddyvdh, i have same problem  you find the solution?

**Comment by AArmandoPerez** [comment]
> Hola@FreddyvdhEn la interfaz de usuario, puedes obtener entradas archivadas agregando un `active = false`filtro. ¿Intentaste hacer lo mismo usando la API?

yes but dont work     var products = await repository.Query().Where(x => x.Active, OdooOperator.EqualsTo, false).ToListAsync();
